### DISCUSSION TITLE
New  features  idea

### DISCUSSION THREAD

**Discussion by Anubi5x0** [Syslifters/sysreptor#282] [General]
If i have pdf template , why not  creat  tool or plugin to convert pdf template or  designs ?

**Comment by aronmolnar** [comment]
Hi Abdel,

Can you elaborate this in more detail and explain what you want to achieve?  
What format would you like to convert into what format?

**Reply by Anubi5x0** [reply]
Develop a plugin or tool for SysReptor that automates  PDF pentest reports into SysReptor-compatible template reports. The plugin would parse PDF content, extract relevant sections (e.g., findings, severity, recommendations), and generate a structured Markdown template that can be imported into SysReptor for further customization or report generation. , so without make it manul , i hope it's clear for you .  thanks for interst

**Reply by aronmolnar** [reply]
Thank you for this idea.

You can export a project using the settings page (this will be downloaded as `tar.gz`) and then use the [unpackarchive command](https://docs.sysreptor.com/cli/utils/unpackarchive/) from the `reptor` CLI tool to create a JSON or a TOML structure.

You could make your adaptions and repack it using the [packarchive command](https://docs.sysreptor.com/cli/utils/packarchive/) to repack it to `tar.gz` and import it again into SysReptor.

Does this fulfill your requirements?
